### ISSUE TITLE
Implement presentation of reduplication and initial change

### ISSUE THREAD

**Issue by aarppe** [UAlbertaALTLab/cree-intelligent-dictionary#585]
While we now show separately information about preverbs, as retrieved from the linguistic analysis, reduplication and initial change are now excluded. We would want to show information about at least reduplication similar to preverbs. The treatment of initial change requires separate consideration.

Reduplication presents a tricky task, as the light/weak and heavy/strong reduplication are realized in multiple ways, based on the following phonology, the meaning of light/weak as indicating _ongoing, continuing_ and of heavy/strong as expressing _intermittent, repeatedly; again and again; here and there_ stays the same across the different realizations. Consider e.g. the following analysis:

```
ê-ma-miyo-mâh-mâmawi-na-nikamot
PV/e+RdplW+PV/miyo+RdplS+PV/mamawi+RdplW+nikamow+V+AI+Cnj+3Sg
```

This should result in the following presentation:

Preverb: ê- <img width="30" alt="image" src="https://user-images.githubusercontent.com/15003407/102447086-7e603c80-3fec-11eb-9cf8-9c928d8f981e.png">
Reduplication: ma- <img width="30" alt="image" src="https://user-images.githubusercontent.com/15003407/102447086-7e603c80-3fec-11eb-9cf8-9c928d8f981e.png">
Preverb: miyo- <img width="30" alt="image" src="https://user-images.githubusercontent.com/15003407/102447086-7e603c80-3fec-11eb-9cf8-9c928d8f981e.png">
Reduplication: mâh- <img width="30" alt="image" src="https://user-images.githubusercontent.com/15003407/102447086-7e603c80-3fec-11eb-9cf8-9c928d8f981e.png">
Preverb: mâmawi- <img width="30" alt="image" src="https://user-images.githubusercontent.com/15003407/102447086-7e603c80-3fec-11eb-9cf8-9c928d8f981e.png">
Reduplication: na- <img width="30" alt="image" src="https://user-images.githubusercontent.com/15003407/102447086-7e603c80-3fec-11eb-9cf8-9c928d8f981e.png">

Then, the <img width="30" alt="image" src="https://user-images.githubusercontent.com/15003407/102447086-7e603c80-3fec-11eb-9cf8-9c928d8f981e.png"> button for any case of light/weak reduplication should show _ongoing, continuing_  and for any case of heavy/strong reduplication: _intermittent, repeatedly; again and again; here and there_.

Matching the tags `RdplW+` and `RdplS+` to their meanings should be straight-forward, but representing the appropriate realization in a non-hacky, language-independent way can be a challenge.

**Comment by aarppe** [comment]
As an interim measure, I suppose reduplication could be represented with templates, i.e. `Ca-/ay-` or `[ckmnpstwy]a-/ay-` for light/weak and `Câh-/âh-` or `[ckmnpstwy]âh/âh-`for heavy/strong.

**Comment by aarppe** [comment]
Also, initial change could be indicated by just that label, `Initial change`. The `(i)` button next to it could lay out the various changes, i.e.

```
a -> ê
i -> ê
o -> wê
ê -> iyê
â -> iyâ
î -> â / iyî
ô -> iyô 
```
I suppose we could add the above as the content for an special dictionary entry for _Initial Change_, and have a reference to a grammar page somewhere.

**Comment by aarppe** [comment]
The English definitions for light and heavy reduplication, according to Arok, are:

**Light/Weak**: _ongoing, continuing_
https://sapir.artsrn.ualberta.ca/cree-dictionary/search?q=sa-

**Heavy/Strong**: _intermittent, repeatedly, iteratively; again and again; here and there_
https://sapir.artsrn.ualberta.ca/cree-dictionary/search?q=s%C3%A2h

**Comment by nienna73** [comment]
Implemented in #859
